tests/installation/logs_from_installation_system.pm

```diff
@@ -48,8 +48,8 @@ sub run {
         # set serial console for xen
         set_serial_console_on_vh('/mnt', '', 'xen') if (get_var('XEN')                      || check_var('HOST_HYPERVISOR', 'xen'));
         set_serial_console_on_vh('/mnt', '', 'kvm') if (check_var('HOST_HYPERVISOR', 'kvm') || check_var('SYSTEM_ROLE',     'kvm'));
-        set_dom0_mem('/mnt')    if (get_var('REGRESSION') && (get_var('XEN') || check_var('HOST_HYPERVISOR', 'xen')));
-        set_pxe_efiboot('/mnt') if check_var('ARCH', 'aarch64');
+        adjust_for_ipmi_xen('/mnt') if (get_var('REGRESSION') && (get_var('XEN') || check_var('HOST_HYPERVISOR', 'xen')));
```

Comment:
Will this adjustion overwrite earlier xen serial console setting in set_serial_console_on_vh for xen?

Reply:
No, it will not.